rebar.config

```diff
@@ -5,11 +5,22 @@
 
 {extra_src_dirs, ["examples"]}.
 
-{project_plugins, [erlfmt]}.
+{project_plugins, [erlfmt, rebar3_ex_doc]}.
 
-{edoc_opts, [{preprocess, true}]}.
+{ex_doc, [
+    {extras, [
+          {"CHANGELOG.md", #{title => "Changelog"}},
+          {"README.md", #{title => "Overview"}},
+          {"LICENSE", #{title => "License"}}
+    ]},
+    {main, "README.md"},
+    {homepage_url, "https://github.com/grisp/grisp"},
```

Comment:
I don't know where this is used in ExDoc, but it could also point to `https://grisp.org` if it makes sense.

Reply:
Yeah, that makes more sense